File: Retos/Reto #29 - EL CARÁCTER INFILTRADO [Fácil]/typescript/Qv1ko.ts
infiltrator("The infiltrated character", "the infiltrat3d-character")

function infiltrator(base: string, str: string) {

    let characters: string[] = []

    if (base.length === str.length) {

        for (let i = 0; i < base.length; i++) {

            if (base[i] !== str[i]) {
                characters.push(str[i])
            }

        }

    }

    console.log(characters)

}

File: Retos/Reto #29 - EL CARÁCTER INFILTRADO [Fácil]/java/Qv1ko.java
import java.util.ArrayList;

public class Qv1ko {

    public static void main(String[] args) {
        infiltrating("The infiltrated character", "The infiltr4ted-character");
    }

    private static void infiltrating(String base, String str) {

        ArrayList<String> infiltrators = new ArrayList<String>();

        if (base.length() == str.length()) {

            for (int i = 0; i < base.length(); i++) {

                if (base.charAt(i) != str.charAt(i)) {
                    infiltrators.add(Character.toString(str.charAt(i)));
                }

            }

        }

        System.out.println(infiltrators);

    }

}
